web_src/js/features/repo-settings.js

```diff
@@ -54,7 +54,8 @@ export function initRepoSettingSearchTeamBox() {
         $.each(response.data, (_i, item) => {
           const title = `${item.name} (${item.permission} access)`;
           items.push({
```

Comment:
```suggestion
          items.push({
```
Seems like an oversight.
(Regarding why two lines: Sometimes I hate GitHub. They introduced yet again a reviewing bug.)

Reply:
What are you suggesting here?

Reply:
Removing the variable

Reply:
Already removed, thanks.